extensions/support-page/main.ts

```diff
@@ -6,7 +6,7 @@ export default class SupportPageMainExtension extends LensMainExtension {
       parentId: "help",
       label: "Support",
       click: () => {
-        this.navigate("/support");
+        this.navigate();
```

Comment:
Interesting. This couldn't be more simple :) Makes sense if your extension only has one page but if there are more than one page you would probably give them all ids (rather than giving one the `""` id)? Or maybe one of the pages could be considered the "top" page. Is this the idea? Guess it's up to the extension developer...

Reply:
Yes, I guess it's up to developer to manage proper ID-s. We could introduce though at least `page.id` validation on registration for uniqueness pages within extension.